### PR TITLE
[1LP][RFR] Upgrade scripting and test fixes for repo changes

### PR DESCRIPTION
**Pull request by lcouzens**
Updated tests and scripts to use new repo files.
REQUIRES: Gitlab MR609
{{pytest: -v -k "test_upgrade_single_inplace or test_appliance_update"}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 988e301ba0e90a05c64e1051eef9dcc355bf78fa:

`cfme/tests/cli/test_appliance_update.py`:
- :red_circle: Line 1:1: **F401** *'tempfile' imported but unused*
- :red_circle: Line 74:12: **F821** *undefined name 'cfme_data'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1927bd484dca8352371cbd2faf412d2cf935ca3d:

`cfme/scripting/appliance.py`:
- :red_circle: Line 11:1: **F401** *'tempfile' imported but unused*

`cfme/tests/cli/test_appliance_update.py`:
- :red_circle: Line 1:1: **F401** *'tempfile' imported but unused*
- :red_circle: Line 74:12: **F821** *undefined name 'cfme_data'*

`cfme/tests/test_db_migrate.py`:
- :red_circle: Line 1:1: **F401** *'tempfile' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit aca38622e0b3f54b1fb0c3a49720e7f6731cd934:

`cfme/scripting/appliance.py`:
- :red_circle: Line 11:1: **F401** *'tempfile' imported but unused*

`cfme/tests/test_db_migrate.py`:
- :red_circle: Line 1:1: **F401** *'tempfile' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/utils/appliance/db.py [hunk 1]
```diff
@@ -60,21 +60,22 @@ def is_partition_extended(self):
...
-            ssh.run_command("umount /repo")
-            ssh.run_command("lvreduce --force --size -9GB /dev/mapper/VG--CFME-lv_repo")
-            ssh.run_command("mkfs.xfs -f /dev/mapper/VG--CFME-lv_repo")
-            ssh.run_command("lvextend --resizefs --size +9GB /dev/mapper/VG--CFME-lv_var")
-            ssh.run_command("mount -a")
-            result = ssh.run_command("df -h")
-            self.logger.info("File systems after extending the DB partition:\n{}"
-                             .format(result.output))
-            ssh.run_command("touch /var/www/miq/vmdb/.db_partition_extended")
+        if self.appliance.version < '5.8.1':
```
Comment (digitronik) [inline cfme/utils/appliance/db.py]: Question:
can we still require it? We have 5.8 break branch and in master we supporting 5.9+.

**Comment by lcouzens** [comment]
Tests are running but looks like a yaml change has not been merged from 3 weeks back. MR 609

**Comment by lcouzens** [comment]
Remaining failures are all sprout issues.

File: cfme/fixtures/appliance.py [hunk 2]
```diff
@@ -89,12 +89,14 @@ def temp_appliance_preconfig_funcscope(appliance, pytestconfig):
...
 def temp_appliance_preconfig_funcscope_upgrade(appliance, pytestconfig):
-    stream_digits = ''.join([i for i in get_stream(appliance.version) if i.isdigit()])
+    split_version = (str(appliance.version).split("."))
     with sprout_appliances(
             appliance,
             config=pytestconfig,
             preconfigured=True,
-            stream="downstream-{}z".format(int(stream_digits) - 1)  # n-1 stream for upgrade
+            stream="downstream-{}{}z".format(
+                split_version[0], (int(split_version[1]) - 1)
```
Comment (mshriver) [inline cfme/fixtures/appliance.py]: :+1:

**Pull request merged**